### PR TITLE
add config for prisma language server

### PR DESCRIPTION
**Pull request by nathanaelcunningham**
adding a config setup for the prisma language server

### CODE REVIEW

File: lua/lspconfig/prismals.lua [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        prismaFmtBinPath = "";
+      }
+    };
+    root_dir = function(fname)
+      return util.root_pattern(".git")(fname);
+    end
+  };
+  docs = {
+    description = [[
+https://github.com/theia-ide/typescript-language-server
```
Comment (ii14) [inline lua/lspconfig/prismals.lua:25]: Linked to typescript-language-server?

File: lua/lspconfig/prismals.lua [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  default_config = {
+    cmd = {bin_name, "--stdio"};
+    filetypes = {"prisma"};
+    settings = {
+      prisma = {
+        prismaFmtBinPath = "";
+      }
+    };
+    root_dir = function(fname)
+      return util.root_pattern(".git")(fname);
```
Comment (ii14) [inline lua/lspconfig/prismals.lua:20]: I think this could just be `root_dir = util.root_pattern(".git")`

Comment (nathanaelcunningham) [reply lua/lspconfig/prismals.lua:20]: that actually was not working for me

File: lua/lspconfig/prismals.lua (hunk 1 above)
Comment (nathanaelcunningham) [reply lua/lspconfig/prismals.lua]: this is updated

File: lua/lspconfig/prismals.lua [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+configs[server_name] = {
+  default_config = {
+    cmd = {bin_name, "--stdio"};
+    filetypes = {"prisma"};
+    settings = {
+      prisma = {
+        prismaFmtBinPath = "";
+      }
+    };
+    root_dir =util.root_pattern(".git");
```
Comment (lithammer) [inline lua/lspconfig/prismals.lua:19]: Are there any other relevant files that can act as root directory? For example `package.json` or some Prisma specific configuration file?

Comment (nathanaelcunningham) [reply lua/lspconfig/prismals.lua]: i added package.json. From what i can tell, there isnt another root file that would be useful.

**Comment by lithammer** [comment]
Squash the commits, then I'll merge.

**Comment by nathanaelcunningham** [comment]
> Squash the commits, then I'll merge.

done

**Comment by mjlbach** [comment]
We can't do this re: prisma-fmt, the binary needs to be on path, or it will crash prisma.

**Pull request merged**